setup.cfg

```diff
@@ -1,3 +1,4 @@
 [flake8]
 ignore = E266, W503
-per-file-ignores = */api.py:F401, */__init__.py:F401
+per-file-ignores = */undo/*:H101, */api.py:F401, */__init__.py:F401,
```

Comment:
Could we split it up into multiple lines and comment why `apptools.undo` is ignored?
If it can reference the issue that mentions deprecating `apptools.undo` that would be idea.

It has occurred in the past that _someone_ would go and assume things are temporarily ignored and wasted time linting something that is about to die anyway.

Reply:
That makes sense.  I don't know that there is an issue open currently, but I will open one and mention it here

Reply:
Thank you!